templates/setup.html

```diff
@@ -13,25 +13,34 @@
   <script>
     $( document ).ready(function() {
       $('#id_context_tags').select2({
-        tokenSeparators: [',', '\n']
+        tokenSeparators: [',', '\n'],
       });
       $('#grouping_link').click(dumpFormData);
       $('#edit_grouping_link').click(dumpFormData);
       $('#add_tag').click(dumpFormData);
-
       $('#id_context_tags').on('select2:select', function(e) {
         var data = e.params.data;
-
         if (e.params.data.id == '{{ curricular_focus_id }}') {
           $('#curricular-focus-modal').modal();
         }
       });
-
       $('#curricularFocusInput').change(function() {
         $('#id_curricular_focus').val($(this).val());
       });
-    });
 
+      /** Hide `course` field when `re_setup` is 'True'. */
+      function hideCourse() {
```

Comment:
Oh no, this is not good. You should do this during page rendering with django template if.
sth like
```
{% if not re_setup %}
couse html here
{% endif %}
```

BTW name is not verbose enough. It suggests that you always hide course elements. Correct way is to remove if statement from this method or rename function.